check_http_json.rb

```diff
@@ -398,7 +398,7 @@ def sanity_check(options)
 if options[:perf].is_a?(Array) then
     p = []
     options[:perf].each do |x|
-        if json_flat[x] then
+        if json_flat.has_key(x) then
```

Comment:
Good catch, thanks & sorry, I didn't test the performance metrics part and left this stupid typo, but I believe it makes sense to change it here as well. Fixed.